src/test/java/io/valkey/springframework/data/valkey/examples/ValkeyApplication.java

```diff
@@ -28,7 +28,7 @@ public class ValkeyApplication {
 
 	public static void main(String[] args) {
 
-		LettuceConnectionFactory connectionFactory = new LettuceConnectionFactory();
+		ValkeyGlideConnectionFactory connectionFactory = new ValkeyGlideConnectionFactory();
```

Comment:
There are only 2 examples in this folder, and only this one can be used with Valkey GLIDE (the other is a reactive example, which GLIDE does not support).